pkg/ccl/backupccl/schedule_exec.go

```diff
@@ -99,19 +99,33 @@ func (e *scheduledBackupExecutor) executeBackup(
 	}
 	backupStmt.AsOf = tree.AsOfClause{Expr: endTime}
 
-	log.Infof(ctx, "Starting scheduled backup %d", sj.ScheduleID())
-
 	// Invoke backup plan hook.
 	hook, cleanup := cfg.PlanHookMaker(ctx, "exec-backup", txn.KV(), sj.Owner())
 	defer cleanup()
 
+	planner := hook.(sql.PlanHookState)
+	currentClusterID := planner.ExtendedEvalContext().ClusterID
+	currentDetails := sj.ScheduleDetails()
+	if currentClusterID != currentDetails.ClusterID {
+		log.Infof(ctx, "pausing backups Schedule %d and updating Cluster ID in schedule details. persisted cluster ID %d is different than current cluster ID %d",
```

Comment:
nit: I'd flip and shorten this; "schedule last run by cluster %s, pausing until manually resumed"